services/velero/7.2.2/helmrelease/helmrelease.yaml

```diff
@@ -27,25 +27,3 @@ spec:
       name: velero-7.2.2-d2iq-defaults
```

Comment:
let's bump the velero version to 7.2.3 to ensure the upgrade

Reply:
i ll do that in https://github.com/mesosphere/kommander-applications/pull/2975 shortly after this PR.